var/spack/repos/builtin/packages/trilinos/package.py

```diff
@@ -715,7 +733,8 @@ def define_tpl_enable(cmake_var, spec_var=None):
             gotype = spec.variants['gotype'].value
             options.extend([
                 define('Tpetra_INST_DOUBLE', True),
-                define('Tpetra_INST_INT_INT', gotype == 'int'),
+                define('Tpetra_INST_INT_INT', gotype == 'int' or
+                      '+epetra' in spec and '+xpetra' in spec),
```

Comment:
I'm not sure Trilinos 13 allows multiple instantiations per build. Should this be a conflict? If left as-is, I'd add a set of parenthesis around the `and` fo improved readability.